axsis.magix.py

```diff
@@ -84,10 +140,23 @@ def on_completed(self):
         pass
 
 
+def create_argparser():
+    parser = argparse.ArgumentParser(description='Start AXSIS Magix client for selected controller.')
+    parser.add_argument('--host', default="localhost", help='Host of controller')
+    parser.add_argument('--port', default=50000, type=int, help='Port of controller')
+    parser.add_argument('--debug', action="store_true", help='Enable debug log')
```

Comment:
Add MODE here?

Reply:
What is the purpose?

Reply:
Nevermind, I was thinking it might be useful to set the mode at startup, but if can specify it with each message - this is even more fine grain.